egs/multi_en/s5/conf/ami_ihm/mfcc.conf

```diff
@@ -0,0 +1,4 @@
+--use-energy=false
+--sample-frequency=16000
```

Comment:
The feature configuration file is ideally same for all the databases, unless you are doing some very specific pre-processing. Are you planning to add this feature ? If not, I would strongly recommend just maintaining one `conf/mfcc.conf` file.